lpc55_sign_bin/src/main.rs

```diff
@@ -199,7 +276,12 @@ fn main() -> Result<()> {
             }
             let cfg: CertConfig = certs.try_into_config()?;
             let root_certs = pad_roots(read_certs(&cfg.root_certs)?)?;
-            let debug_settings = DebugSettings::default();
+
+            let debug_settings = std::fs::read_to_string(&debug_settings_cfg)?;
+            let debug_settings = toml::from_str(&debug_settings).context(format!(
```

Comment:
Nit: use `with_context` for expensive error message operations (this one allocates and fills in a String, for instance)